en/mulesoft/modules/ROOT/pages/ms_composer_using_actions_in_a_flow.adoc

```diff
@@ -193,8 +193,8 @@ When mapping source data fields:
 
 * The source field must be a data pill with a fixed list of values.
 * The target field must be an enumerated xref:ms_composer_custom_expression_editor.adoc#data-types[data type] field or any string field. 
-* If any other unmapped value from the source system is found, the default mapping specifies the value in the target system . If you do not specify a default value, the original value from the source field is passed to the target field.
-* The mapping is stored at the organization level and can be reused across flows.
+* If any other unmapped value from the source system is found, the default mapping specifies the value in the target system. If you do not specify a default value, the original value from the source field is passed to the target field.
+* The mapping is stored at the organization level and can be reused across flows. You don't need to perform any extra steps when reusing mappings across different flows compared to reusing mappings within the same flow.
```

Comment:
Can you shorten this to something like "You can reuse mappings both within the same flow and across different flows without performing extra steps." ?

Reply:
What about this to compare that there are no extra steps if you want to reuse mappings across different flows?

No additional steps are required to reuse mappings across different flows compared to within the same flow.